src/joint.dia.graph.js

```diff
@@ -5,21 +5,21 @@ joint.dia.GraphCells = Backbone.Collection.extend({
 
     cellNamespace: joint.shapes,
 
+    activeBatches: {},
+
     initialize: function(models, opt) {
 
-        //In order to avoid multiple sort during toFront and toBack deep operations
-        //The change:z event is disabled during batch:start and reactivated when all the batches have stopped
-        var activeBatches = 0;
-        this.on('batch:start', function(e) { if ((e.batchName === 'to-front') || (e.batchName === 'to-back')) { ++activeBatches; }});
-        this.on('batch:stop', function(e) {
-            if (activeBatches > 0 && ((e.batchName === 'to-front') || (e.batchName === 'to-back'))) {
-                this.sort();
-                --activeBatches;
-            }
-        }, this);
+        this.on('batch:start', function(e) { this.activeBatches[e.batchName] = (this.activeBatches[e.batchName] || 0) + 1; }, this);
```

Comment:
Without setting a default, 'undefined' would be the default.
With this modification not only undefined but anything that is convertible to false like ``null``, ``0`` and ``''`` would be reset to default, but not ``1`` for example.
What is the expected behavior?